README.md

```diff
@@ -16,14 +17,27 @@ Grab via Maven:
 </dependency>
 ```
 or Gradle:
-```groovy
-repositories {
-    mavenCentral()
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
-}
+Add it in your root build.gradle at the end of repositories:
+
 
-compile 'com.michaelpardo:activeandroid:3.1.0-SNAPSHOT'
+```groovy
+allprojects {
```

Comment:
Why `allprojects`? What if somebody wants to configure this for a specific module? or subprojects?